### ISSUE TITLE
[$150]THRIVE - How Beginner Designers Can Win UI Design Challenges in the Topcoder Community

### ISSUE THREAD

**Issue by hasanozie** [topcoder-platform/thrive-submit#160]
Track Category: Design

[Hasanozie - How Beginner Designers Can Win UI Design Challenges in the Topcoder Community.docx](https://github.com/topcoder-platform/thrive-submit/files/3998133/Hasanozie.-.How.Beginner.Designers.Can.Win.UI.Design.Challenges.in.the.Topcoder.Community.docx)

**Comment by kkartunov** [comment]
passed checks

**Comment by Dara-K** [comment]
Great article! I like it a lot but I see it more of a fit for the blog than THRIVE as it's not revealing any  knowledge that is specific to winning a challenge, it's more of a motivational article to guide the new members on Topcoder design way. Would you agree to add it to the blog?

If yes, below are a couple of notes to update:

> A little story about my experience getting to know the Topcoder Community, which began when I joined the Medan Creator Meetup Community, which often held graphic design workshops and began to discuss UI Design Challenges in the Topcoder Community where the workshop was run by several Veteran Members of the Topcoder Community such as Fajarmln, abedavera and lunarkid.

This sounds like 2 phrases more than 1. Can you separate them? Also please add when exactly (date) did you join the Medan Meetup Community.

**Stop Comparing Yourself With Veteran Members**
On this section, you might want to add about the veterans, that they were beginners once and learned a lot and grew so they got where you see them - winning so much.
Newbie members can also reach the veterans to ask them questions on how to be a better designer.

**Never Stop Competing in UI Design Challenges**
You might want to add that even when you lose from a design challenge, you can still learn from the winner's design and do better in the future.

> Those are some tips to increase self-confidence that you can apply when competing in the UI Design Challenge in the Topcoder Community. so nothing can stop you from taking part in various Challenges in the Topcoder Community. 

Could you rephrase this so it's not repeating twice the "Challenges in the Topcoder Community"?

Thank you.

**Comment by hasanozie** [comment]
Hi @Dara-K 
It doesn't matter if this article will be added to the topcoder blog,
I will correct this article according to the advice you gave.
thanks.

**Comment by Dara-K** [comment]
Thank you!

**Comment by hasanozie** [comment]
Hi @Dara-K  ,
Below is the google docs link for this article
Thanks :)

https://drive.google.com/file/d/1Oy9eeHZBE6ceYg9fsUvE9zrZQfOTrowV/view?usp=sharing

**Comment by Dara-K** [comment]
Hi @hasanozie 
Thanks for the updates. I have added a couple of comments in the document. Please review and confirm here when you did.

**Comment by hasanozie** [comment]
Hi @Dara-K 
I have corrected the article according to your suggestion. is there anything else that needs to be fixed?

**Comment by Dara-K** [comment]
Thank you, it's good to go. Well done!

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30112836 has been created for this ticket.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30112836 has been updated - it has been assigned to hasanozie.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30112836<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```
